### ISSUE TITLE
Alterations of the vaginal microbiome in healthy pregnant women positive for group B Streptococcus colonization during the third trimester

### ISSUE THREAD

**Issue by SvetlanaUP** [waldronlab/BugSigDBcuration#323]
Alterations of the vaginal microbiome in healthy pregnant women positive for group B Streptococcus colonization during the third trimester – Sarah Shabayek – BMC Microbiology
https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-022-02730-8

**Comment by Nana-Habiba** [comment]
@SvetlanaUP May i curate this paper.
My first curation is marked review: https://bugsigdb.org/Study_932

**Comment by Nana-Habiba** [comment]
@SvetlanaUP May i still claim this paper, My second curation should be marked for a review
:https://bugsigdb.org/Study_1027

**Comment by Nana-Habiba** [comment]
I am done with this: @Nana-Habiba you are assigned #317

**Comment by Nana-Habiba** [comment]
@SvetlanaUP Thanks

**Comment by Nana-Habiba** [comment]
@SvetlanaUP @cmirzayi Needs review, Third curation: https://bugsigdb.org/Study_1054

**Comment by Scholarpat** [comment]
Hi @SvetlanaUP, 
I've reviewed and completed this curation.

Thank you

**Comment by SvetlanaUP** [comment]
https://bugsigdb.org/Study_1054 reviewed.

**Comment by SvetlanaUP** [comment]
@Scholarpat please leave a comment what has been corrected. Thanks!

**Comment by Scholarpat** [comment]
Hi @SvetlanaUP,

Here are the changes I made to this curation:

- I adjusted the group 1 definition and source description to make them more descriptive.

- I edited the antibiotic exclusion to include only the time frame: 6 months.

- Signatures highlighted in orange, which were initially curated as C. urealyticum and P. anaerobius, have been changed to Corynebacterium urealyticum and Peptostreptococcus anaerobius respectively.

Thank you.
